src/emulator/auth/operations.ts

```diff
@@ -215,18 +215,17 @@ function lookup(
         }
       }
     }
-    return {
-      kind: "identitytoolkit#GetAccountInfoResponse",
-      users,
-    };
   } else {
     assert(reqBody.idToken, "MISSING_ID_TOKEN");
     const { user } = parseIdToken(state, reqBody.idToken);
     users.push(redactPasswordHash(user));
   }
   return {
     kind: "identitytoolkit#GetAccountInfoResponse",
-    users,
+
+    // Drop users property if no users are found. This is needed for Node.js
```

Comment:
nit: this isn't technically dropping the property; the response will still have a "users" property with value `undefined`. That means it'll still show up in `Object.keys(response)`.

Change the wording of this comment to be more explicit that the field is being set to undefined if the user array is empty

Reply:
JSON doesn't have an undefined type and `JSON.stringify` will just drop that field.

Reply:
I forgot the responses are `JSON.stringify`'d